### PR TITLE
Add new upsert endpoint for link checker api

### PR DESCRIPTION
**Pull request by SebAshton**

### CODE REVIEW

File: lib/gds_api/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module GdsApi
-  VERSION = '49.6.0'.freeze
+  VERSION = '49.7.0'.freeze
```
Comment (thomasleese) [inline lib/gds_api/version.rb]: I think this should be in a separate commit/PR.

**Comment by SebAshton** [comment]
@thomasleese yep just adding that now

**Comment by SebAshton** [comment]
@thomasleese should be GTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 49.7.0
```
Comment (thomasleese) [inline CHANGELOG.md]: I would leave the version heading out at the moment as, although unlikely, another PR may get merged before making the release which changes this to a major update. The release PR can add the heading.
